x-pack/plugins/security_solution/public/common/components/exceptions/viewer/exceptions_viewer_items.tsx

```diff
@@ -81,21 +78,13 @@ const ExceptionsViewerItemsComponent: React.FC<ExceptionsViewerItemsProps> = ({
             exceptions.length > 0 &&
             exceptions.map((exception, index) => (
               <MyFlexItem data-test-subj="exceptionItemContainer" grow={false} key={exception.id}>
-                {index !== 0 ? (
```

Comment:
Removes the `OR` between exception item cards as it no longer appears in new designs.